### PR TITLE
Removed And operation

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

File: ngraph/test/backend/auto_broadcast.in.cpp [hunk 1]
```diff
@@ -104,74 +104,6 @@ void check_auto_bcast(
     }
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, auto_bcast_binary_elementwise)
```
Comment (tomdol) [inline ngraph/test/backend/auto_broadcast.in.cpp:107]: Those tests were unrelated to the operation removed in this PR. Should they really go away?

Comment (ilyachur) [reply ngraph/test/backend/auto_broadcast.in.cpp:107]: In this test we use only opset0 operations... Also we use And operation.

File: ngraph/test/runtime/CMakeLists.txt [hunk 2]
```diff
@@ -31,8 +31,10 @@ set (SRC
     performance_counter.hpp
     dynamic/dynamic_backend.cpp
     dynamic/dynamic_backend.hpp
+    op/and.cpp
+    op/and.hpp
     op/atan2.cpp
-    op/atan2.hpp
+    op/atan2.hp
```
Comment (GlebKazantaev) [inline ngraph/test/runtime/CMakeLists.txt:37]: I think it's typo

Comment (ilyachur) [reply ngraph/test/runtime/CMakeLists.txt]: Fixed

**Pull request merged**